docs/reviewing-release-prs.md

```diff
@@ -0,0 +1,176 @@
+
+```
+git diff @metamask/<package-name>@<current-version>..HEAD -- packages/<package-name>
+```
+
+You'll reference these changes in upcoming steps.
+
+## 2. Determine impact of changes
+
+Now that you know what sort of changes are being made, the next step is look through the version bumps in the release PR and verify the impact of those changes so that it can be properly communicated.
+
+First, it's helpful to understand what sort of changes are worth communicating. While a release could certainly ship with a list of every single change made to the code in this repository, this would be needless. The people who will end up using the packages in this monorepo — consumers — only care about that part of code they can actually see and work with. So when it comes to understanding impact, this is what you should pay attention to:
+
+- Changes to interfaces
+- Changes to behavior
+- Changes to runtime dependencies
+- Changes to consumer-facing documentation
+
+Conversely, you generally should not need to worry about:
+
+- Changes to internal tooling
+- Changes to developer-only dependencies
+- Changes to contributor-only documentation
+
+With that in mind, there are four categories of changes that determine the level of impact.
+
+### Breaking changes
+
+If a consumer upgrades a package to a new version without making any modifications to their code, that version is "breaking" if it causes:
+
+- An error at runtime
+- An error at compile time (e.g., a TypeScript type error)
+- An error at install time (e.g., the consumer's package manager reports a Node version incompatibility)
+- A surprising difference in behavior
+
+Given this, there are many ways a change could be regarded as breaking:
+
+- Changing a function or method to throw an error
+- Adding a required argument to a function or method
+- Adding a property to a TypeScript type
+- Narrowing the type of a property in a TypeScript type
+- Narrowing the type of an argument in a function
+- Adding a parameter to a TypeScript type
+- Removing a parameter from a TypeScript type
+- Upgrading a runtime dependency to a version which causes any of the above
+- Removing an export from the package
+- Renaming the package
+- Bumping the Node version
+- Bumping the NPM version
+
+### Additions
+
+Minimally, this category could cover any actions that extend the surface area of a package in some way:
+
+- Adding an optional property to an existing type
+- Adding an optional argument to an existing function or method
+- Adding a new export to a package
+
+However, it is important to remember that consumers care about capabilities first and interfaces second. So, some additions could also be more simply regarded as changes. For instance:
+
+- Adding a new dependency
+
+### Removals
+
+In most cases, removing something from a package will cause a breaking change, and thus should be categorized as such. There are some rare instances, however, when this is not the case:
+
+- Removing a runtime dependency
```

Comment:
Along similar lines, this is not something I'd view as a "removal" at all. Nothing is being removed from the package API when a dependency is removed.

Reply:
In the context of deciding on what the SemVer version should be, I don't think we need to have this section here. Maybe we can remove this and let it be covered by the "Everything else" section

Reply:
> In the context of deciding on what the SemVer version should be, 

Hmm. The original goal of this section was not solely to decide what the SemVer version is, nor to decide how changes get categorized in the changelog. It was just to discuss different kinds of changes that you might see when preparing a release — what to pay attention to and what not to pay so much attention to — so that it was easier to figure out how to decide on a SemVer version and how to categorize changes in the changelog.

That said, I probably inadvertently mapped these categories map more to Keep a Changelog categories than SemVer categories. And in any case it seems confusing to have three separate ways of categorizing changes in this document.

I'll think about this some more.

Reply:
I updated the section on categories to fit the intersection between SemVer and Keep a Changelog. I removed this particular line but if there were any items that I wanted to call out as not belonging in a category, I marked them with an X.